Description: Remove PySide requirement, since python-pyside does not register itself correctly
Author: Dain Nilsson <dain@yubico.com>
Forwarded: no
--- a/setup.py
+++ b/setup.py
@@ -44,8 +44,9 @@
     entry_points={
         'gui_scripts': ['neoman=neoman.__main__:main']
     },
-    install_requires=['PySide', 'pycrypto'],
-    yc_requires=['ctypes', 'qt'],
+    install_requires=['pycrypto'],
+    yc_requires=['ctypes'],
+    packages=['neoman', 'neoman.model', 'neoman.view', 'neoman.yubicommon', 'neoman.yubicommon.setup', 'neoman.yubicommon.ctypes', 'neoman.yubicommon.qt'],
     cmdclass={'executable': executable, 'qt_resources': qt_resources('neoman')},
     classifiers=[
         'License :: OSI Approved :: BSD License',
